src/package/packageCreate.ts

```diff
@@ -64,10 +57,12 @@ export function createPackageDirEntry(project: SfProject, options: PackageCreate
     packageDir.default = !packageDirs.find((pd: PackageDir) => pd.default);
   }
 
-  packageDir.package = packageDir.package || options.name;
-  packageDir.versionName = packageDir.versionName || pkgUtils.DEFAULT_PACKAGE_DIR.versionName;
-  packageDir.versionNumber = packageDir.versionNumber || pkgUtils.DEFAULT_PACKAGE_DIR.versionNumber;
-  packageDir.versionDescription = packageDir.versionDescription || options.description;
+  packageDir.package = !packageDir.package ? options.name : packageDir.package;
+  packageDir.versionName = !packageDir.versionName ? pkgUtils.DEFAULT_PACKAGE_DIR.versionName : packageDir.versionName;
+  packageDir.versionNumber = !packageDir.versionNumber
+    ? pkgUtils.DEFAULT_PACKAGE_DIR.versionNumber
+    : packageDir.versionNumber;
+  packageDir.versionDescription = !packageDir.versionDescription ? options.description : packageDir.versionDescription;
```

Comment:
Shouldn't all of these be a version of this (nullish coalescing)?
`packageDir.package ??= options.name;`

Reply:
I'll look